src/cr/cube/cubepart.py

```diff
@@ -66,7 +66,7 @@ def cube_index(self):
         return self._cube.cube_index
 
     @lazyproperty
-    def cube_is_mr_by_itself(self):
+    def cube_is_mr_aug(self):
```

Comment:
Seeing the first word as `cube`, and the `lazyproperty` on top, automatically directs me to thinking that it's some kind of an object. I'd probably prefer the `is_cube_mr_aug`, if it's a relatively easy refactor.

That said, if a cube has `is_mr_aug`, why can't a cube-part have the same name for the property? Class membership should be enough to determine if it's in cube or cube-part. But then again, does a "is mr aug" make sense in the cube-part context?

Probably just the first of the two paragraphs makes sense (to actually act on), but I'm being verbose on purpose, as the nature of the work is such.